snapcraft/internal/elf.py

```diff
@@ -62,7 +60,6 @@ def ldd(path: str, ld_library_paths: List[str]) -> Dict[str, str]:
 
     env = os.environ.copy()
     env["LD_LIBRARY_PATH"] = ":".join(ld_library_paths)
-    logger.debug(f"invoking ldd with ld library paths: {ld_library_paths!r}")
```

Comment:
We had many issues in the past when calling ldd with variations of LD_LIBRARY_PATH